web/src/lib/components/faces-page/people-card.svelte

```diff
@@ -54,6 +58,7 @@
       <ContextMenu on:outclick={() => (showContextMenu = false)}>
         <MenuOption on:click={() => onHideFaceClicked()} text="Hide face" />
         <MenuOption on:click={() => onChangeNameClicked()} text="Change name" />
+        <MenuOption on:click={() => onSetBirthDateClicked()} text="Set date of birth" />
```

Comment:
```suggestion
        <MenuOption on:click={() => dispatch('set-birth-date', person)} text="Set date of birth" />
```

Reply:
Sure about this? All the others are simple one-liner methods. Or should I inline the others as well?